packages/create-rspack/index.js

```diff
@@ -45,7 +45,8 @@ yargs(hideBin(process.argv))
 				message: "Project template",
 				choices: [
 					{ title: "react", value: "react" },
-					{ title: "react-ts", value: "react-ts" }
+					{ title: "react-ts", value: "react-ts" },
+					{ title: "vue", value: "vue" }
```

Comment:
maybe we should use "vue3" here

Reply:
Currently vue3 is the default version of vue, I don't think we need to point out the vue version, vue2 is the one that needs to point out

Reply:
I'm not a vue user, it that's the case then it's ok for me